core/site.go

```diff
@@ -500,3 +500,7 @@ func (site *Site) Run(stopC chan struct{}, interval time.Duration) {
 		}
 	}
 }
+
+func (site *Site) Name() string {
```

Comment:
Wenns unbedingt sein muss dann `GetTitle`

Reply:
Ich bin für alternativen offen. Zur not könnte man glaube ich auch nen cast auf die `Site` struct versuchen aber das ist auch doof

Reply:
eine andere Alternative wäre das man `core.Site` statt `site.API` dem MQTT.Run übergeben aber ich weiß nicht ob das eine Option ist